src/Native/FactorizationMachineNative/FactorizationMachineCoreSSE.cpp

```diff
@@ -8,10 +8,10 @@
 #include <limits>
 #include <pmmintrin.h>
 
-// Compute the output value of the field-aware factorization, as the sum of the linear part and the latent part. 
+// Compute the output value of the field-aware factorization, as the sum of the linear part and the latent part.
```

Comment:
Given the difficulty of reading AVX code, please add a comment:
// This function implements Algorithm 1 in [https://github.com/wschin/fast-ffm/blob/master/fast-ffm.pdf](https://github.com/wschin/fast-ffm/blob/master/fast-ffm.pdf)
.

Please add the same line to SSE and FMA counterparts.